app/helpers/pledgegroups_helper.rb

```diff
@@ -0,0 +1,2 @@
+module PledgegroupsHelper
+end
```

Comment:
I think you could delete this file and it'd still be ok (no need to create helper files that do nothing)